### DISCUSSION TITLE
Will copilot be getting updated to use the "GPT 3.5" model

### DISCUSSION THREAD

**Discussion by raesene** [community/community#40680] [Copilot Conversations]
### Select Topic Area

Question

### Body

A new project from OpenAI, called ChatGPT apparently makes use of an updated GPT3 model, which has improved performance on things like code generation and conversion. A quick comparison between ChatGPT and Copilot on language conversion seemed to show that ChatGPT has better functionality there (albeit the interface isn't great for coding)

Is there a plan for Copilot to get upgraded to use this new model?

**Comment by WinkelCode** [comment]
I did some experimenting with ChatGPT and it's really good, GitHub Copilot is already impressive, but ChatGPT once again takes it to another level. Could anyone from the Copilot team chime in how it looks like with upgrades to the Copilot model, is this something we can expect to get at some point?

Also, it would be super neat if there was a chat window to ask Copilot questions directly, especially for shell scripting, I usually put a "scratchpad" file in my project directory when I quickly need to ask Copilot to come up with a command to do something. For example, the other day I needed to convert a video to .mp4 and boost its volume, it having been ages since I used ffmpeg the last time, Copilot gave me the right command via the aforementioned scratchpad file. Doing this in a chat, or something specific to the terminal, would be really nice.
